src/scripts/genEventPipe.py

```diff
@@ -437,7 +437,7 @@ def generateEventPipeImplFiles(
             return false;
     }}
 
-    *(T *)(buffer + offset) = value;
+    memcpy(buffer + offset, (char *)&value, sizeof(T));
```

Comment:
Minor: is the (char *) cast necessary?  I would probably prefer either (void *) or no cast at all unless it makes a difference in warning or behavior.  Not necessary to fix.  Mostly just curious.

Reply:
I had originally though that it would be required to convert from T*.  It does not appear to be required.